app/code/Magento/SalesRule/Controller/Adminhtml/Promo/Quote/Save.php

```diff
@@ -39,6 +46,7 @@ public function __construct(
         $this->timezone =  $timezone ?? \Magento\Framework\App\ObjectManager::getInstance()->get(
             TimezoneInterface::class
         );
+        $this->dataPersistor = $dataPersistor;
```

Comment:
Please add ` $dataPersistor` as an argument to constructor

Reply:
### It is changed.